src/server/game/Entities/Creature/Trainer.cpp

```diff
@@ -21,6 +21,9 @@
 #include "Player.h"
 #include "SpellInfo.h"
 #include "SpellMgr.h"
+#include "WorldSession.h"
```

Comment:
sort the includes :)

Reply:
@Aokromes pls, they wont be needed if author follows our suggesion for moving